server/src/main/java/org/opensearch/env/Environment.java

```diff
@@ -73,6 +73,12 @@ public class Environment {
         Property.NodeScope
     );
     public static final Setting<String> PATH_LOGS_SETTING = new Setting<>("path.logs", "", Function.identity(), Property.NodeScope);
+    public static final Setting<String> HTTP_PROTOCOL_SETTING = new Setting<>(
```

Comment:
Is this setting needed? The security plugin can pass test settings to testClusters' OpenSearchNode with `systemProperty <propertyKey> <propertyValue>` w/o adding any need cluster settings. This setting would also duplicate a setting from the security plugin which is used to determine whether `https` should be used.

Reply:
I dont think so. Let me remove it.

Reply:
Edit: We did not since we build each node with the system properties not from the configs. Removed.